keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensor.java

```diff
@@ -510,6 +510,9 @@ public DoubleTensor expInPlace() {
     public DoubleTensor minusInPlace(DoubleTensor that) {
         if (that.isScalar()) {
             tensor.subi(that.scalar());
+        } else if (this.isScalar()) {
+            // Temporary fix: Scalar can't be subtracted by tensor inplace
```

Comment:
Can we drop the comment about it being a "Temporary fix"? Can we document the method as having this behaviour and that is the fix. Later when we change the behaviour we'll update the method docs appropriately.